### DISCUSSION TITLE
FR: record time stamp

### DISCUSSION THREAD

**Discussion by sokai** [iSoron/uhabits#993] [Feature Requests]
Is it possible to record the time stamp for a numeric counter/measurable habit, the (last) change time?

Thanks and BR!

**Comment by VA-MMXXII** [comment]
This could be useful, especially if the user is targeting a general time of day deadline or spacing or interval of occurences.

**Comment by alextio** [comment]
I also want this to be added. 

Right now, I'm trying to wake up early as a habit, and the lack of timestamp support makes this app hard to rely upon.

**Comment by anffray** [comment]
Hi,
I am using the app to track my daily habits as well and I would like to compare my habits over the years.
It would be super usefull to have the timestamp for each counter.
Do you think it could be easily added to the app ? 
Thanks all !

**Comment by herrbrixter** [comment]
Duplicate of #1313
